cpp/include/cuvs/neighbors/common.hpp

```diff
@@ -90,15 +90,6 @@ struct index_params {
   cuvs::distance::DistanceType metric = cuvs::distance::DistanceType::L2Expanded;
   /** The argument used by some distance metrics. */
   float metric_arg = 2.0f;
-  /**
```

Comment:
Why is it removed from here? Is it only that we can repeat the docstring in each subclass? If that is the case, why not keep this and add the following with the docstring in the subclasses?
```c++
/** docstring */
using cuvs::neighbors::index_params;
```

Reply:
`add_data_on_build` is added separately to the index params for IVF-Flat and IVF-PQ. It is not used in other algorithms.